src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -217,13 +218,15 @@ const createTransaction = (
       dispatch(closeSnackbarAction({ key: pendingExecutionKey }))
     }
 
-    dispatch(enqueueSnackbar(errorMsg))
+    dispatch(enqueueSnackbar({ key: err.code, message: errorMsg, options: { persist: true, variant: 'error' } }))
 
-    const executeDataUsedSignatures = safeInstance.methods
-      .execTransaction(to, valueInWei, txData, operation, 0, 0, 0, ZERO_ADDRESS, ZERO_ADDRESS, sigs)
-      .encodeABI()
-    const errMsg = await getErrorMessage(safeInstance.options.address, 0, executeDataUsedSignatures, from)
-    console.error(`Error creating the TX - an attempt to get the error message: ${errMsg}`)
+    if (err.code !== REJECT_CONFIRM_TX_ERROR_CODE) {
```

Comment:
If I reject the attempt to sign an off-chain tx, this condition is true as the `tryOffchainSigning` throws an error without `code`

Reply:
Thanks for the feedback, was very useful, found also more bugs with notifications on proccessTransaction that I fixed as well